tests/protocols/test_calibrationmodel.py

```diff
@@ -143,18 +144,18 @@ def fit_si(spe_sil):
         profile="Pearson4", candidates=cand, no_fit=False, vary_baseline=False
     )
     df = fitres.to_dataframe_peaks().sort_values(by="height", ascending=False)
-    difference = abs(df.iloc[0]["center"] - si_peak)
-    assert difference < 1e-2, f"Si peak found at {df.iloc[0]['center']}"
+    difference = abs(df.iloc[0]["position"] - si_peak)
```

Comment:
not sure if position in the cwa have same meaning as position in the lm-fit. i suspect that in cwa by position they mean centroid while in lm-fit they mean mode (i believe)

Reply:
CWA means the peak ; if lm-fit position means the mode , then this is the peak . 
Using the center as before was clearly identified as an issue by @nicocopez & @rportelar